packages/react-liveness/src/components/FaceLivenessDetector/shared/Hint.tsx

```diff
@@ -90,7 +94,14 @@ export const Hint: React.FC<HintProps> = ({ hintDisplayText }) => {
 
   const getInstructionContent = () => {
     if (isStartView) {
-      return defaultToast(hintDisplayText.hintCenterFaceText, true);
+      return (
+        <>
+          <VisuallyHidden role="alert">
+            {hintDisplayText.hintCenterFaceInstructionText}
+          </VisuallyHidden>
+          {defaultToast(hintDisplayText.hintCenterFaceText, true)}
```

Comment:
Curious if there is reasoning to use an imperative API here? As far as i can tell `defaultToast` can just be a React element

Reply:
No particular reason at this time, can update